### ISSUE TITLE
[Bug]: Injection `Symbol(SidebarContext)` not found. Component must be used within `Sidebar`

### ISSUE THREAD

**Issue by DanielSBoba** [radix-vue/shadcn-vue#882]
### Reproduction

https://codesandbox.io/p/devbox/cranky-rhodes-hms3pk?workspaceId=87418a45-b88e-427a-85aa-d226284d3610

### Describe the bug

<template>
    <Sidebar></Sidebar>
</template>

gives

Injection `Symbol(SidebarContext)` not found. Component must be used within `Sidebar`

### System Info

```bash
Windows, Brave
```

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by DreKwasi** [comment]
@DanielSBoba faced the same issue. Wrap your component with `SidebarProvider`. This should fix the issue

**Comment by Soviut** [comment]
The error message should really spell this out.

**Comment by DreKwasi** [comment]
If you understand a bit of React you can peep at the original ShadCN docs. Explains everything including the props and hooks(in Vue's case composables)

**Comment by Soviut** [comment]
@DreKwasi That's not very helpful. Needing to reference another project's docs is not how we solve this problem. Updating the docs here and improving the error message so that it clearly states that the SidebarProvider is missing is what's needed.

**Comment by DreKwasi** [comment]
Agreed. It's just a temporary approach till this project's maintainers make the updates.

**Comment by janniksohn** [comment]
The same Issue occurs when using the `useSidebar` method from the utils. Is there a Fix for that?

See Line 25:
https://stackblitz.com/edit/shadcn-vue-sidebar?file=app.vue

**Comment by demiro** [comment]
I am having the same issue:

do we use Context providers in Vue as well?

**Comment by zernonia** [comment]
Please check the latest document to use Sidebar. You need to wrap `SidebarProvider` at the root of your application, then use `useSidebar` would work 😁
